demo/c/picovoice_demo_file.c

```diff
@@ -52,29 +119,27 @@ int main(int argc, char *argv[]) {
     const char *library_path = argv[1];
     const char *porcupine_model_path = argv[2];
     const char *keyword_path = argv[3];
-    const float porcupine_sensitivity = (float) atof(argv[4]);
+    const float porcupine_sensitivity = (float) strtod(argv[4], NULL);
     const char *rhino_model_path = argv[5];
     const char *context_path = argv[6];
-    const float rhino_sensitivity = (float) atof(argv[7]);
+    const float rhino_sensitivity = (float) strtod(argv[7], NULL);
```

Comment:
We're getting a double from `strtod` then casting to float. Could we just use `strtof` instead?